mix.exs

```diff
@@ -5,13 +5,17 @@ defmodule LoggerSplunkBackend.Mixfile do
     [
       app: :logger_splunk_backend,
       version: "2.0.0",
```

Comment:
Should we bump the version here? And I think the change in Elixir version is a breaking change, so we might even have to go to 3.0.0?

Reply:
Given that this is only changing to support our new CI provider, I wasn't planning to do a release. There's no new functionality here.